source/before-integrating/index.html.md.erb

```diff
@@ -0,0 +1,60 @@
+
+If you’re integrating multiple services, you should [create a configuration for each service you’re integrating](/before-integrating/#create-a-configuration-for-each-service-you-re-integrating). 
+
+Make sure you scope your services according to the [GOV.UK Service Standard guidance](https://www.gov.uk/service-manual/service-standard) on how users think and what they need to do. Find more [information on scoping your service](https://www.gov.uk/service-manual/design/scoping-your-service). 
+
+## Create a configuration for each service you’re integrating
+
+GOV.UK One Login is an OpenID Connect (OIDC) provider. An OIDC ‘relying party’ is an app that outsources its user authentication function to an identity provider, which in this instance is GOV.UK One Login.
+
+To interact with GOV.UK One Login, you must first [register each of your services with GOV.UK One Login as a relying party](/integrate-with-integration-environment/manage-your-service-s-configuration/#register-your-service-to-use-gov-uk-one-login). You need to do this for each of the services that you want to integrate with GOV.UK One Login.
+
+Part of the service’s configuration is the `client-id`, which is a unique identifier that GOV.UK One Login uses to identify your services. Each service should have a distinct `client-id`.
+
+GOV.UK One Login uses `client-id` for: 
+
+* retrieving configurations
+* event auditing
+* performance analytics 
+* fraud prevention and data sharing
+
+When you are ready to set up your service’s configuration, follow the guidance on how to [manage your service’s configuration with GOV.UK One Login](/integrate-with-integration-environment/manage-your-service-s-configuration/#manage-your-service-s-configuration-with-gov-uk-one-login).
+
+### Why you should use a specific configuration for each service
+
+You must use individual configurations for each of your services to get the following benefits: 
+
+* GOV.UK One Login can provide a report detailing each service’s performance including success rates and volumes
+* if a service gets compromised, the outage is limited to an individual service
+* fraud analysis can use information about how users access different services and better profile and detect fraudulent behaviour
+* the support team has more context regarding user activity and can better support with user queries
+* users have visibility on all available services and their activity logs which makes navigation to specific services easier and suspicious activities more likely to be detected  
+
+If you are not using individual configurations for each of your services, be aware that:
+
+* fraudulent activity cannot be easily profiled and detected
+* you cannot get service specific analytics and we cannot generate these retrospectively in case you decide not to have individual configurations from the start
+* your users cannot benefit from simplified navigation and service specific information through their user journey
```

Comment:
```suggestion
If you do not use individual configurations for each of your services, GOV.UK One Login cannot:

* monitor or recognise fraud as effectively 
* give you service specific analytics, nor generate this retrospectively if you later switch to individual configurations 
* provide your users with simplified navigation or service specific information to improve their user journey
```

Some suggestions to make this active instead of passive but I think may need a check from Phil - what do you think?

Reply:
@sabrinaharris I didn't know how to add a suggestion to your suggestion, so I edited your - so just to check with you, what you think of this version? I have discussed the changes with Phil as well.

Reply:
Looks good! 👍